mobly/controllers/android_device.py

```diff
@@ -917,6 +917,9 @@ def load_snippet(self, name, package):
         client. E.g. `name='maps'` attaches the snippet client to
         `ad.maps`.
       package: string, the package name of the snippet apk to connect to.
+      snippet_client_configs: snippet_client_v2.Config, the configuration
```

Comment:
simply `config` or `client_config` should be fine?
It's pretty clear that we are dealing with snippet here..

also the args are eventually passed to the snippet server affecting the launch stage. So it's not exactly "client config"

Reply:
How about `snippet_configs`?

I have thought about this problem, but the name `config` is a bit confusing, because in this module there is another kind of config, i.e. controller configs. So I want to make it more clear. Also, you're right `client_config` is also not accurate, so I changed it `snippet_configs`.

Reply:
Between the type hints and docstring, it's obvious which config this is right?

The method's name literally includes "snippet" in it..

Reply:
Ok, make sense. Changed `snippet_configs` -> `configs`.